README.md

```diff
@@ -71,55 +71,10 @@ Once the backend is up and running, the XxllncZGWBundle can be configured. To en
         *   Select the user and edit the password.
 
 2.  **Security Group**
-    *   Add the scopes for the Default Anonymous in the Security Group.
+    *   Scopes have been added for the xxllnc user. You can view and adjust the scopes of the user via the following steps:
```

Comment:
You could add an url to the 'werkinstructie' for this at some point
see: https://conduction.atlassian.net/browse/DIT-167

Reply:
Heb hier een nieuw issue voor gemaakt: XZ-165